skyxplore-ui-test/src/test/java/com/mentoring/pageobject/FactoryPage.java

```diff
@@ -19,4 +31,19 @@ public static String getPageUrl() {
     FactoryPage waitUntilPageLoads() {
         return this;
     }
+
+    public void clickExpansionItem() {
+        waitUntilClickable(expansionItem).click();
+    }
+
+    public FactoryPage clickCex01ManufacturingButton() {
+        waitUntilClickable(cex01ManufacturingButton).click();
+        return this;
+    }
+
+    @Step("Asserting if expansion item under manufacturing is visible.")
```

Comment:
No real assertion happens here, it's just checking

Reply:
It is a hard assertion called from EquipmentTest. Could you please check it once more? (I found similar solution in other pages classes (e.g. IndexPage, CharacterSelectionPage).